### DISCUSSION TITLE
Isn't the "LXC" term misleadingly used?

### DISCUSSION THREAD

**Discussion by krzakx** [tteck/Proxmox#1713] [Q&A]
Hello to all.
I am pretty new to containers world and having in mind following definition of [LXC ](https://en.wikipedia.org/wiki/LXC) term makes me thining about this as **system** container.
While in those great scripts! repo https://tteck.github.io/Proxmox/ 

Following entries:
* Home Assistant Container LXC
* Home Assistant Core LXC

does not really are an **system** container.
But from my best understanding at least
* Home Assistant Core LXC
is an installation on top of Proxmoxx.

Please correct me if I am wrong.

**Comment by tteck** [comment]
Maybe reading these will give you a better understanding of a LXC, https://linuxcontainers.org/lxc/introduction/ and https://pve.proxmox.com/wiki/Linux_Container

**Reply by krzakx** [reply]
Thank you for quick replay.

Does _Home Assistant Container LXC_ means that Home Assistant Core is installed inside a Docker container, which in turn (Docker) is installed inside LXC where this LXC in turn reside on Proxmoxx VE OS (Debian flavoured distro) ?

**Reply by tteck** [reply]
The Home Assistant Container LXC is as follows:

1) Information is gathered and a LXC is created within Proxmox, not on Proxmoxx VE OS. This container serves as an isolated environment where you can run various applications.
2) Dependencies are installed to facilitate the Home Assistant Container installation method. This includes essential components like Python, Docker, runlike, and more. Seen here https://github.com/tteck/Proxmox/blob/main/install/homeassistant-install.sh

**Reply by tteck** [reply]
So, in this architecture, Home Assistant Core is ultimately running inside a Docker container, which is running within an LXC container, and all of this is happening within Proxmox VE

**Reply by Arkaqius** [reply]
Hi,
I thought that HA Supervisor is not supported under LXI, or we just talk about core only?

https://www.home-assistant.io/more-info/unsupported/lxc

Arek

**Reply by tteck** [reply]
I haven't come across anyone else mentioning "Supervisor" except for you.

**Reply by Arkaqius** [reply]
Ok, thanks for make it clear.

**Reply by tteck** [reply]
Apologies if my previous response seemed unkind. To clarify, we are referring to Home Assistant Core without the Supervisor component.
